paper/paper.md

```diff
@@ -0,0 +1,329 @@
+`cloc` tool [@cloc]; blank and comment lines are excluded from the
+estimate. Test coverage was estimated using the `coveralls` output for `Pint`
+and `astropy.units` and using the `codecov.io` output for `unyt`.
+
+| Library                        | `unyt`         | `astropy.units` | `Pint`     |
+|--------------------------------|----------------|-----------------|------------|
+| Lines of code                  | 5128           | 10163           | 8908       |
+| Lines of code excluding tests  | 3195           | 5504            | 4499       |
+| Test Coverage                  | 99.91%         | 93.63%          | 77.44%     |
+
+We offer lines of code as a very rough estimate for the "hackability" of the
+codebase. In general, smaller codebases with higher test coverage are have fewer
+defects [@Lipow1982; @Koru2007; @Gopinath2014]. This comparison is somewhat
+unfair in favor of `unyt` in that `astropy.units` only depends on NumPy and
+`Pint` has no dependencies, while `unyt` depends on both `sympy` and NumPy. Much
+of the reduction in the size of the `unyt` library can be attributed to
+offloading the handling of algebra to `sympy` rather than needing to implement
+the algebra of unit symbols directly in `unyt`. For potential users who are wary
+of adding `sympy` as a dependency, that might argue in favor of using `Pint` in
+favor of `unyt`.
+
+![A benchmark comparing the time to apply units to lists and NumPy `ndarray` instances. Each test is shown for three different sizes of input data, including inputs with size 3, 1,000, and 1,000,000. The black lines at the top of the bars indicate the sample standard deviation. The $T_{\rm numpy}$ time is calculated by benchmarking the time to perform `np.asarray(data)` where `data` is either a `list` of an `ndarray`.](apply.png)
+
+## Astropy.units
+
+The `astropy.units` subpackage provides a `PrefixUnit` class, a `Quantity` class
+that represents both scalar and array data with attached units, and a large
+number of predefined unit symbols. The preferred way to create `Quantity`
+instances is via multiplication with a `PrefixUnit` instance. Similar to `unyt`,
+the `Quantity` class is implemented via a subclass of the NumPy `ndarray`
+class. Indeed, in many ways the everyday usage patterns of `astropy.units` and
+`unyt` are similar, although `unyt` is not quite a drop-in replacement for
+`astropy.units` as there are some API differences. The main functional
+difference between `astropy.units` and `unyt` is that `astropy.units` is a
+subpackage of the larger `astropy` package. This means that depending on
+`astropy.units` requires depending on a large collection of astronomically
+focused software, including a substantial amount of compiled C code. For users
+who are not astronomers or do not need the observational astronomy capabilities
+provided by `astropy`, depending on all of `astropy` just to use `astropy.units`
+may be a tough sell.
```

Comment:
I don't have a suggestion to replace, but this is a bit colloquial and could be read slightly negatively (and I don't think that's your intention.)